src/ingestor.ts

```diff
@@ -171,13 +184,10 @@ export default async function ingestor(req) {
     let pinned;
     let receipt;
     try {
-      const { address, sig, ...restBody } = body;
-      const ipfsBody = {
-        address,
-        sig,
-        hash: id,
-        ...restBody
-      };
+      const ipfsBody = writer[type].getIpfsBody
```

Comment:
What's the purpose of this check ? Afaik, none of the writer exposes a `getIpfsBody` function.